raft/raft_test.go

```diff
@@ -4138,8 +4203,8 @@ func (nw *network) drop(from, to uint64, perc float64) {
 }
 
 func (nw *network) cut(one, other uint64) {
-	nw.drop(one, other, 2.0) // always drop
-	nw.drop(other, one, 2.0) // always drop
+	nw.drop(one, other, 1.0) // always drop
+	nw.drop(other, one, 1.0) // always drop
```

Comment:
Why do we change this?

`raft/raft_test.go:4201:42: (*network).drop - perc always receives 1.0 (1)`

Seems like now we only use `1.0` for `nw.drop` (CI was complaining :0)

Reply:
It was a fix for what I thought was a typo. Dropping 200% of messages doesn't really make sense.

I didn't think it would trigger a lint warning. Reverted.